### PR TITLE
Add docstring to merge cards task

### PR DESCRIPTION
**Pull request by renceInbox**

### CODE REVIEW

**Review by jeremi — CHANGES_REQUESTED** [review]
please use the restructuredText format for the docstring

**Comment by renceInbox** [comment]
Updated @jeremi with additional docstring to other functions.

File: card_generator/api/v1/cards/views.py [hunk 1]
```diff
@@ -36,7 +39,12 @@ class CardViewSet(ModelViewSet):
         detail=True,
     )
     def render(self, request, **kwargs):
-        """Generate a card from a template with the provided values."""
+        """
+        Generate a card from a template with the provided values.
+        :param request: Request object
+        :param kwargs: Extra data to pass
```
Comment (jeremi) [inline card_generator/api/v1/cards/views.py:45]: `Extra data to pass` Does not help much the user to know what it is.

**Review by jeremi — CHANGES_REQUESTED** [review]

File: card_generator/api/v1/cards/views.py (hunk 1 above)
Comment (renceInbox) [reply card_generator/api/v1/cards/views.py]: update. what do you think with the new description @jeremi ?

**Review by jeremi — APPROVED** [review]

**Pull request merged**